doc/classes/ProjectSettings.xml

```diff
@@ -2449,6 +2450,12 @@
 			If [code]true[/code], performs a previous depth pass before rendering 3D materials. This increases performance significantly in scenes with high overdraw, when complex materials and lighting are used. However, in scenes with few occluded surfaces, the depth prepass may reduce performance. If your game is viewed from a fixed angle that makes it easy to avoid overdraw (such as top-down or side-scrolling perspective), consider disabling the depth prepass to improve performance. This setting can be changed at run-time to optimize performance depending on the scene currently being viewed.
 			[b]Note:[/b] Depth prepass is only supported when using the Forward+ or Compatibility rendering method. When using the Mobile rendering method, there is no depth prepass performed.
 		</member>
+		<member name="rendering/driver/line_drawing/width" type="float" setter="" getter="" default="1.0">
+			Render hardware-based 3D lines using the specified value as a thickness (in pixels). Wire thickness does not depend on distance from the camera. This setting affects all lines drawn in 3D, including gizmos in the editor and debug draw options. This can be used to make debug drawing more visible, particularly when using a high-resolution viewport or on hiDPI displays.
```

Comment:
```suggestion
			The thickness of all hardware-based 3D lines (in pixels). This thickness does not depend on distance from the camera. This setting affects all lines drawn in 3D, including gizmos in the editor and debug draw options. This can be increased to make debug drawing more visible, particularly when using a high-resolution viewport or on hiDPI displays.
```
I personally feel like there's something redundant about "_This can be increased to make debug drawing more visible, particularly when using a high-resolution viewport or on hiDPI displays_" that could be reduced, but I have no suggestions at the moment.